README.md

```diff
@@ -40,7 +40,7 @@ if (!isAutomateSupported(chainId)) {
 3. Instantiate Automate using your signer:
 
 ```typescript
-const automate = new AutomateSDK(chainId, signer);
+const automate = await AutomateSDK.create(chainId, signer);
```

Comment:
we will need to release a new major version for this, since it has breaking changes